network.go

```diff
@@ -720,6 +720,15 @@ func (n *network) CreateEndpoint(name string, options ...EndpointOption) (Endpoi
 			ep.iface.mac = mac
 		}
 	}
+	// This may change depending how Aliases are transmitted by daemon
```

Comment:
If we do, the alias will be specific to the each endpoint object (by the way - there is already aliases and myAliases - which will cause potential confusion with people used to linux terminology).
In common implementation, IP aliases are assigned to loopback, and not a specific interface. it doesn't seem like this is possible from a plugin.